specification/contosowidgetmanager/Contoso.WidgetManager/tspconfig.yaml

```diff
@@ -11,21 +11,24 @@ options:
   "@azure-tools/typespec-autorest":
     azure-resource-provider-folder: "data-plane"
     emitter-output-dir: "{project-root}/.."
-    examples-directory: "examples"
+    examples-directory: "{project-root}/examples"
```

Comment:
FYI - [issue#3182](https://github.com/Azure/typespec-azure/issues/3182)

Reply:
I'm OK working around this for now but do we know if we can get some of these to be defaults so we don't have to specify them in all the tspconfig files?